bower.json

```diff
@@ -31,13 +31,13 @@
     "angular-bootstrap": "^0.13.0",
     "angular-bootstrap-slider": "~0.0.6",
     "angular-load": "~0.4.1",
-    "angular-sanitize": "~1.3.0",
-    "angular-ui-router": "0.2.15",
+    "angular-sanitize": "~1.5.0",
+    "angular-ui-router": "^0.2.18",
     "angular-ui-tinymce": "0.0.18",
     "bootstrap-form-components": "https://github.com/Rise-Vision/bootstrap-form-components.git",
     "bootstrap-sass-official": "~3.2.0",
     "component-storage-selector": "https://github.com/Rise-Vision/component-storage-selector.git",
-    "component-subscription-status": "https://github.com/Rise-Vision/component-subscription-status.git",
+    "common-header": "https://github.com/Rise-Vision/common-header.git",
     "rv-angular-bootstrap-colorpicker": "https://github.com/Rise-Vision/angular-bootstrap-colorpicker.git",
     "rv-common-i18n": "https://github.com/Rise-Vision/common-i18n.git",
     "rv-common-style": "https://github.com/Rise-Vision/common-style.git",
```

Comment:
@stulees you can actually remove the common-i18n and common-style references here; those repos are no longer used.

Also, in Apps we simplify the dependencies and necessary resolutions by just allowing Common-Header to define the Angular version and other library versions (including jquery, bootstrap, angular plugins, etc). Just a suggestion for keeping the bower.json file clean.

Thanks!